.circleci/config.yml

```diff
@@ -199,37 +199,94 @@ jobs:
+              sudo apt-key adv --keyserver packages.microsoft.com --recv-keys 52E16F86FEE04B979B07E28DB02C46DF417A0893
+              sudo apt-get install apt-transport-https
+              sudo apt-get update && sudo apt-get install azure-cli
           name: "Installing packages"
       - run:
           command: |
+              echo $EXTRACT_ENDPOINT_STORAGE_DOMAIN
+              echo $EXTRACT_ENDPOINT_STORAGE_ACCOUNT
+              echo $EXTRACT_ENDPOINT_STORAGE_KEY
+              echo $EXTRACT_ENDPOINT_CONTAINER
+              az storage container create --name energuide-extracted-data --connection-string 'DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;'
+          name: "Create azure container"
+      - run:
+          command: |
+              cd etl
               python3 -m venv ~/.venv
               . ~/.venv/bin/activate
               pip install -r requirements.txt
               pip install -e .
-          name: "Installing dependencies"
+          name: "Installing dependencies for energuide"
       - save_cache:
-          key: "deps1-{{ .Branch }}-{{ checksum \"requirements.txt\" }}"
+          key: "deps1-{{ .Branch }}-{{ checksum \"etl/requirements.txt\" }}"
           paths:
             - ~/.venv
       - run:
           command: |
+              cd extract_endpoint
+              python3 -m venv ~/.venv
+              . ~/.venv/bin/activate
+              pip install -r requirements.txt
+              pip install -e .
+          name: "Installing dependencies for endpoint"
+      - save_cache:
+          key: "deps2-{{ .Branch }}-{{ checksum \"extract_endpoint/requirements.txt\" }}"
+          paths:
+            - ~/.venv
+      - run:
+          command: |
+              cd extract_endpoint
+              . ~/.venv/bin/activate
+              python3 src/extract_endpoint/endpoint.py
+          background: true
+          name: "Start flask endpoint app"
+      - run:
+          command: |
+              cd etl
+              . ~/.venv/bin/activate
+              python3 src/energuide/flask_app.py
+          background: true
+          name: "Start flask etl app"
+      - run:
+          command: |
+              cd etl
               . ~/.venv/bin/activate
               energuide extract --infile tests/randomized_energuide_data.csv --outfile tests/randomized_energuide_data.zip
-              energuide load  --db_name energuide --collection dwellings --filename tests/randomized_energuide_data.zip
-          name: "Loading data"
+          name: "Extracting data"
+      - run:
+          command: |
+              cd extract_endpoint
+              . ~/.venv/bin/activate
+              extract_endpoint upload ../etl/tests/randomized_energuide_data.zip 2018-03-14
+          name: "Send data to azure, trigger TL process"
+      - run:
+          sleep 5
```

Comment:
Should we wait until #386 is merged in so we can remove this sleep and spin until we get a status that it's done?